client/src/entrypoints/admin/modal-workflow.js

```diff
@@ -105,9 +92,10 @@ function ModalWorkflow(opts) {
       // if response contains an 'html' item, replace modal body with it
       if (useDialog) {
         self.body.innerHTML = response.html;
+        self.body.classList.remove('w-opacity-0');
       } else {
-        self.body.html(response.html);
-        self.container.modal('show');
+        self.container.show();
+        self.body.innerHTML = response.html;
```

Comment:
If I recall correctly, jquery's `html(...)` function has the hidden feature of executing any `<script>` tags in the HTML content, which `innerHTML` doesn't do. If that is the case, we should check whether any of our modals are relying on inline `<script>`s (I suspect they are...) and ensure we reinstate that behaviour.

Reply:
Yeah jQuery does do that. It’s not too hard to replicate with vanilla JS but I think we could just stick to jQuery’s `.html` for the time being (and add a comment that this is for good reason).